src/components/SequenceInput/SequenceInput.js

```diff
@@ -95,7 +95,7 @@ export default function SequenceInput({ onSaveClick }) {
             <LowerThirdContainer>
               <TimecodeInput
                 style={{ margin: '10px 0' }}
-                title="Timecode IN"
+                title="Bauchbinde IN in der Szene"
```

Comment:
The title is a little bit hard to understand, because of the two "in". Maybe you can change it to something like this: 
"Bauchbinde IN (innerhalb der Szene)"

Reply:
Very good. Had a hard time to find the right wording.